sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetry.java

```diff
@@ -55,6 +56,7 @@
 import static com.azure.cosmos.implementation.guava25.base.Preconditions.checkNotNull;
 
 public class ClientTelemetry {
+    public final static String VM_ID_PREFIX = "vmId:";
```

Comment:
I see there is a place we are trying to replace `vmId:` to `vmId_`, why not use vmId_ from beginning?

Reply:
Won't fix - the "vmId: " prefix is coming from ClinetTelemetry - same prefix used in .Net and expected to be used in Kusto etc. Changing the prefix would technically be breaking. For ClientMetrics I just have to get rid of the ":" because metric names can't use all characters (to simplify using metric names in Uris). This is a micrometer constraint.